<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>662</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>662</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>926</width>
    <height>662</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>911</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>100</horstretch>
      <verstretch>100</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Vk</string>
     </attribute>
     <widget class="QWebView" name="webView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>891</width>
        <height>561</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
     <widget class="QLabel" name="label_movie">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>300</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>411</width>
        <height>291</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>90</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineCorectX">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineCorectY">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>160</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Options</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditAddAlbum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>431</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ссылка на альбом. http://vk.com/album.....            Потом нажать Enter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAddGroup">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>431</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ссылка на группу. http://vk.com/.....            Потом нажать Enter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAddText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>881</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Текст сообщения. Только латинкие символы ( буквы, цыфры, символы )</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLoadAlbum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Album</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSaveAlbum">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Album</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClearAlbum">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Album</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClearGroup">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>70</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Group</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLoadGroup">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Group</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSaveGroup">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>70</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Group</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>881</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QListWidget" name="listWidgetAlbum"/>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetGropus"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Для остановки нажмите PAUSE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUrl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Url&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>911</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>15</height>
     </size>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Пауза перед вставкой</string>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>label</zorder>
   <zorder>labelUrl</zorder>
   <zorder>checkBox</zorder>
   <zorder>progressBar</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/trig2.png</normaloff>:/trig2.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
